packages/stylex/package.json

```diff
@@ -2,13 +2,28 @@
   "name": "@stylexjs/stylex",
   "version": "0.3.0",
   "description": "A library for defining styles for optimized user interfaces.",
-  "main": "lib/stylex.js",
-  "types": "lib/stylex.d.ts",
+  "main": "./lib/stylex.js",
+  "module": "./lib/es/stylex.js",
+  "types": "./lib/stylex.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/es/stylex.js",
+      "require": "./lib/stylex.js",
+      "types": "./lib/stylex.d.ts"
+    },
+    "./lib/StyleXCSSTypes": "./lib/StyleXCSSTypes.js",
+    "./lib/StyleXSheet": "./lib/StyleXSheet.js",
+    "./lib/StyleXTypes": "./lib/StyleXTypes.js",
```

Comment:
@nmn can we avoid exporting these files?

Reply:
In some rare cases, `StyleXTypes` might be useful to export, but outside of that only `stylex.js` and `stylex-inject.js` are needed.

Reply:
Do you think we need the commonJS exports?

Reply:
removed the `StyleXCSSTypes` and `StyleXSheet` exports

Reply:
looks like `lib/StyleXSheet` export is used in `dev-runtime` package. I will add it back